plane/src/drone/docker/mod.rs

```diff
@@ -40,6 +40,10 @@ pub mod types;
 /// The existence of this label is used to determine whether a container is managed by Plane.
 const PLANE_DOCKER_LABEL: &str = "dev.plane.backend";
 
+fn backend_id_to_container_id(backend_id: &BackendName) -> ContainerId {
+    ContainerId::from(format!("plane-{}", backend_id.to_string()))
+}
```

Comment:
nit: listen to clippy

Reply:
Can we use `From` and/or `into` for this to leverage Rust typing more?

Reply:
I actually think it's a good thing for the conversion to be explicit here. If it was just a `From` conversion, I would expect it to be just re-packing the same string, not modifying it in an application-specific way.